core/object/undo_redo.cpp

```diff
@@ -517,6 +533,9 @@ void UndoRedo::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("has_undo"), &UndoRedo::has_undo);
 	ClassDB::bind_method(D_METHOD("has_redo"), &UndoRedo::has_redo);
 	ClassDB::bind_method(D_METHOD("get_version"), &UndoRedo::get_version);
+	ClassDB::bind_method(D_METHOD("set_max_steps", "max_steps"), &UndoRedo::set_max_steps);
+	ClassDB::bind_method(D_METHOD("get_max_steps"), &UndoRedo::get_max_steps);
+	ADD_PROPERTY(PropertyInfo(Variant::INT, "max_steps"), "set_max_steps", "get_max_steps");
```

Comment:
Should be moved down between methods and signals

Reply:
I'd say around 50 is a pretty sensible limit. By putting "or_greater" users can go above it, anyway.

Reply:
Agreed, the limit is just to scale the property editor when relevant